### ISSUE TITLE
Feature request: Shortcuts to favorite folders

### ISSUE THREAD

**Issue by benedetto** [owncloud/android#867]
In my owncloud file tree I have some folders that I access frequently from my mobile but which are some two or three levels deep within a rather huge folder tree. Navigating to them means basically opening a folder, scroll to somewhere in the middle of the long file list (you can either scroll fast and will pretty surely overshoot beyond the folder you wanted or scroll sloooooow), open the folder you wanted and repeat the above, which is quite some interaction.

Therefore it would be nice if one could define in the app some favorite folders that can then be accessed by some shortcuts or the like to get around all this scrolling and searching.

**Comment by tobiasKaminsky** [comment]
There is something like that in #757: There will be a "file view" called favorites. 
But at the moment I do not know if this just displays all favorites files, ordered by folders, or shows only favorite files in a current folder.
@jancborchardt, please help us here :)

**Comment by jancborchardt** [comment]
The favorites view should be a flat list displaying all favorite folders and files, regardless of where they are. Not only in the current view. :) It’s a separate section.

**Comment by jancborchardt** [comment]
(Same for all the other sidebar entries basically.)

**Comment by BastiCambeo** [comment]
I think this feature has just been released in the owncloud 8.0 core

So support in the android app would be awesome (I have the same problems as the OP)

**Comment by jancborchardt** [comment]
@tobiaskaminsky @davivel how are we doing with the sidebar? :)

**Comment by benedetto** [comment]
@BastiCambeo Yep, it was. However, I wouldn't necessarily want the same favourites on my mobile that I have on my web interface. Therefore I would prefer to have them set separately or at least to be able set additional one just for mobile use.

**Comment by tobiasKaminsky** [comment]
@jancborchardt I have no time for this right now :/
Maybe in march...
@benedetto So you suggest two favorite type? I do not think that this is a good idea as it will be more difficult to use for the user and can easily lead to confusion.

@jancborchardt @davivel Is there a plan in supporting this favorite files of OC8?

Thinking about it... Currently favorite in the app means "keep in sync"... But this is not the same as favorite is meant in OC8. What to do?

**Comment by jancborchardt** [comment]
Would be really really cool to have this. Anyone up for it @AndyScherzinger @tobiasKaminsky @LukeOwncloud @Kernald

**Comment by davivel** [comment]
Sorry, but we need to drop issues from milestone 2.4.0 to get it out next to OC 10 server.

This time we'll move topics to ```backlog``` instead of directly to next release, so that we can schedule 2.5.0 properly.

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are
closing it as outdated. If this is still relevant on a current ownCloud release,
please **reopen** (or open a fresh issue) with up-to-date details — thank you\!
